sonarts-core/src/runner/processRequest.ts

```diff
@@ -25,12 +25,13 @@ import getHighlighting from "./highlighter";
 import getCpdTokens from "./cpd";
 import getSymbolHighlighting from "./symbolHighlighting";
 import { createProgram } from "../utils/parser";
+import getDiagnostics from "./diagnostics";
 
 interface Sensor {
   (sourceFile: ts.SourceFile, program: ts.Program): any;
 }
 
-const sensors: Sensor[] = [getHighlighting, getMetrics, getCpdTokens, getSymbolHighlighting];
+const sensors: Sensor[] = [getHighlighting, getMetrics, getCpdTokens, getSymbolHighlighting, getDiagnostics];
```

Comment:
@saberduck I see the beauty of adding diagnostics as just one more sensor, but I would prefer to not run other sensors if there is any syntax error (theoretically it can prevent from some runtime exceptions), same for sonartsServer